integration/__tests__/app.tests.ts

```diff
@@ -87,6 +113,19 @@ describe("Lens integration tests", () => {
       await app.client.waitUntilTextExists("h2", "Add Cluster");
     });
 
+    describe("protocol app start", () => {
+      it("should handle opening lens:// links", async () => {
+        const res = await open("lens://app/foobar");
+
+        console.log(res);
+
+        await utils.waitForLogsToContain(app, {
```

Comment:
This might not be a good idea. Infinite loop behind this util will cause problems. Could we have few (one?) real handlers that we can test here. For example `lens://app/preferences` which would open Preferences view.

Reply:
How is the infinite loop any worse then for `app.client.waitUntilTextExists`? Sure I can add a single real handler too.

Reply:
I see lot's of integration failures that leave an app instance behind (we can really have just one instance running)... I'm suspecting it's because this loop never dies.

Reply:
having commented out the tests that have this code I still see those instances behind so I doubt that it has to do with that code (plus I have added a fail fast technique to stop the loop when the test ends (from timeout))